ui/src/types/index.tsx

```diff
@@ -12,7 +12,7 @@ export interface Story extends Model {
   title: string;
   content: string;
   current_city: string;
-  year: number;
+  year: string;
```

Comment:
what was the reasoning behind switching this to a string?

Reply:
The endpoint on the backend looks like this: 
![image](https://user-images.githubusercontent.com/19617248/109722229-6bd0ba80-7b7a-11eb-8154-fcfa0f17cdb8.png)
so I just updated the types.

Reply:
So i think its better to keep it int because when stories are returned, year is an int. It is just that in formdata everything is in string format, so even if you send it as an int, it would be received in the backend as a string.

Reply:
Changed it back 👍🏻